monekcheckout/PaymentProcessor.php

```diff
@@ -51,9 +59,13 @@ private function generate_basket_base64($order){
         if (!empty($order_taxes)) {
             $basket['taxes'] = array();
             foreach ($order_taxes as $tax) {
+                $tax_description = isset($tax['description']) ? $tax['description'] : '';
+                if (strlen($tax_description) > 15) {
+                    $tax_description = substr($tax_description, 0, 15) . '...';
+                }
```

Comment:
We have customers chasing this urgently, im currently sat on a call with support can we just push this through rather than holding it up for some basic refactoring? I can make these changes in the next iteration.

Reply:
You're must be aware that code refactoring isn't going to happen unless there is a bug fixed or an update!